### ISSUE TITLE
Marketing task - evaluate effort required

### ISSUE THREAD

**Issue by pmcpinto** [woocommerce/woocommerce-admin#7195]
**Description:** we're assessing the possibility of moving the marketing plugins promotion from the business details step to a task in the store setup task list. Before moving forward with this, we'd like to have an estimate of the effort required to implement this task, as it will require pausing some of the WCPayments work. More info about the design exploration here: pbIJXs-Ta-p2

**Acceptance criteria:** I plan to add it in the upcoming days

**Comment by joshuatf** [comment]
@pmcpinto Below are my estimates for pieces of this task. As a very rough outline of the assumptions on work needed here:

* Remove the extension installation from the onboarding profiler. (3)
* Create a new task and list component, consuming the remote extensions. (8)
* Enhance the remote extensions list to include extension images. (1)

Most of the legwork has already been handled here, so the bulk of this work is around creating and styling the list components used in this new task.

However, this does not account for the "Finish setup" button in the new task.  I left some comments in pbIJXs-Ta-p2 but if we need a way to reliably determine if a plugin has been fully setup and configured, we'll need a special API to do that and this could be an 13 or even 21 in effort.

**Comment by mattsherman** [comment]
> * consuming the remote extensions

What do you mean by consuming the remote extensions? Is the list of extensions hard-coded in the client, or is this driven off of an API/config file?

**Comment by pmcpinto** [comment]
Thanks @joshuatf 

> However, this does not account for the "Finish setup" button in the new task. I left some comments in pbIJXs-Ta-p2 but if we need a way to reliably determine if a plugin has been fully setup and configured, we'll need a special API to do that and this could be an 13 or even 21 in effort.

I'm in favor of simplifying this to avoid increasing the scope of the marketing task. Let me see if we can get feedback from our partners in the upcoming days. We're going to start cycle 14 tomorrow. If we don't have input from them during this week, it will be better to begin working on this on Cycle 15.

**Comment by joshuatf** [comment]
@pmcpinto Will Facebook remain under the "Grow your store" category for the time being?  I'm trying to figure out how we can reuse the WCCOM REST API here.  I believe we can just create another category to handle the task-list based extensions.

![Screen Shot 2021-07-07 at 8.50.16 PM.png](https://images.zenhubusercontent.com/5d7fd4b446611c000130d6f9/2ee4381c-288a-405d-861b-0b27023f28a4)

**Comment by pmcpinto** [comment]
> @pmcpinto Will Facebook remain under the "Grow your store" category for the time being?

No, I asked @ilyasfoo to remove Facebook from the Business Features step in WooCommerce 5.5 (release date: July 13): #7282. Ilyas, just to confirm if we're on track to remove the plugin. Thanks

**Comment by elizaan36** [comment]
Here's the V2 design with updated copy for the task, more context in the card, and groupings for the extensions. Copy is placeholder. More details and link to the Figma in pbIJXs-Ta

![Screen Recording 2021-07-08 at 10 05 00 AM](https://user-images.githubusercontent.com/5121465/124962981-1c8d1180-dfd4-11eb-9a8f-e70c51c135f5.gif)

**Screenshots**
<img width="674" alt="Screen Shot 2021-07-08 at 8 44 16 AM" src="https://user-images.githubusercontent.com/5121465/124951970-b2bb3a80-dfc8-11eb-8a41-104f76094e9c.png">
<img width="686" alt="Screen Shot 2021-07-08 at 8 44 47 AM" src="https://user-images.githubusercontent.com/5121465/124952053-c5357400-dfc8-11eb-8342-e77f88643f5b.png">
<img width="688" alt="Screen Shot 2021-07-08 at 8 46 05 AM" src="https://user-images.githubusercontent.com/5121465/124952268-f3b34f00-dfc8-11eb-9f32-487188db73bb.png">

**Comment by pmcpinto** [comment]
@joshuatf @elizaan36 One thing that went unnoticed when we were thinking about this new task is that each setup task has a help panel with links to documentation. I've just added this to the acceptance criteria

**Comment by mattsherman** [comment]
@pmcpinto @elizaan36 I know Facebook was removed from the OBW. But, it is shown in the mockups for this feature. Should Facebook be included in these new screens?

**Comment by pmcpinto** [comment]
>  Should Facebook be included in these new screens?

No. For now, we don't include Facebook.

> The same goes for Pinterest, which is also shown in the mockups.

The Pinterest extension is still under development. ETA: September

**Comment by pmcpinto** [comment]
@joshuatf @joelclimbsthings just heads up that I've included a new section in the epic description about the event tracking that has to be implemented. Let me know if you have questions. Thanks

**Comment by pmcpinto** [comment]
> @joshuatf @joelclimbsthings just heads up that I've included a new section in the epic description about the event tracking that has to be implemented. Let me know if you have questions. Thanks

@joshuatf  @joelclimbsthings let me know if you need a separate issue about this.

**Comment by mattsherman** [comment]
> Event name: wcadmin_marketing_finish_setup
Description: when the user clicks to finish setup
Event prop:
extension_name: name of the extension

@pmcpinto I'm not sure where the click to finish setup is.

**Comment by pmcpinto** [comment]
> @pmcpinto I'm not sure where the click to finish setup is.
I'm talking about the "Manage" button. Sorry for the confusion. I'm going to change the event name and description to manage, to avoid any misunderstanding.
